### ISSUE TITLE
Won't work with MariaDB

### ISSUE THREAD

**Issue by idexter** [k1LoW/tbls#274]
### What happened

Docs won't generate

### What you expected to happed

Docs should be generated.

### What stacktrace or error message from tbls did you see?

```
DEBUG=1 tbls doc
malformed version: 10.4.10-MariaDB-1:10.4.10+maria~bionic-log:
    github.com/aquasecurity/go-version/pkg/version.Parse
        /Users/k1low/go/pkg/mod/github.com/aquasecurity/go-version@v0.0.0-20201115065329-578079e4ab05/pkg/version/version.go:44
```

### Anything else we need to know?

Looks like the reason is that that library (`github.com/aquasecurity/go-version`) has been updated.

### Environment

- tbls version: `1.47.0`
- Database version: `MariaDB 10.4.10`
- OS: `Ubuntu bionic`
- Shell: `zsh`

**Comment by k1LoW** [comment]
Hi @idexter,  thank you for your report!!

Until now, tbls did not explicitly support MariaDB, but I will do so.
Please wait for the next release ( or use master branch ).

**Comment by k1LoW** [comment]
Released as v1.48.0.

Use `mariadb` instead of `mysql` in the DSN scheme as follows

```
mariadb://dbuser:dbpass@hostname:3306/dbname
```
